app/models/section.rb

```diff
@@ -1,6 +1,8 @@
 class Section < Sequel::Model
   extend ActiveModel::Naming
 
+  set_primary_key [:position]
```

Comment:
So this affects nothing and is just me being picky about being explicit about codifying the primary key the same we do in other models